fusor-ember-cli/app/controllers/review/installation.js

```diff
@@ -90,7 +90,7 @@ export default Ember.Controller.extend({
       // name is fqdn for managed host
       return (this.get("selectedRhevEngine.name"));
     }
-  }.property('selectedRhevEngine', 'engineDomain'),
+  }.property('selectedRhevEngine', 'selectedRhevEngine.name', 'engineDomain'),
```

Comment:
@cfchase, yes, we can add `selectedRhevEngine.is_discovered`. The value shouldn't change unless a different deployment selected the same engine (i.e. discovered_host) and clicked "Deploy". The "Deploy" backend action using Foreman / Sat6 is what changes it from `Host::Discovered` to `Host::Managed` and then it wouldn't appear in the list of engines to be selected anyway.